src/reanimated2/__mocks__/NativeReanimated.js

```diff
@@ -2,7 +2,20 @@ global._setGlobalConsole = (val) => {};
 
 export default {
   installCoreFunctions: () => {},
-  makeShareable: (worklet) => worklet,
+  makeShareable: (worklet) => {
+    var _f = function _f() {
+      return worklet();
+    };
+
+    _f._closure = {};
+    _f.asString = worklet.toString();
+    _f.__workletHash = Math.floor(Math.random() * 10000000000);
+    _f.__location = 'jest tests fixture (1:0)';
+
+    global.__reanimatedWorkletInit(_f);
```

Comment:
Can we rely on this function being defined? AFAIK it comes from the plugin and some folks disable it in test env.

Reply:
Indeed, it's not available. I am mocking it in host project. 
What is your proposal? Provide a mock?

Reply:
I think we can mock it like in [`core.js`](https://github.com/software-mansion/react-native-reanimated/blob/3781d823b597a7afc8005a6ff96f3755e122610e/src/reanimated2/core.js#L6-L8) or just make it noop.

Reply:
alright. This will not break if folks disable it in test env.

Reply:
After review, `__reanimatedWorkletInit` is indeed available in test env (since `core.js` was loaded every time) and is basically noop. However, I moved such globals into dedicated file (if there is any need to mock them in the future)